tests/regression/nokiasrl/nokiasrl_test.py

```diff
@@ -0,0 +1,434 @@
+      ]
+    }
+}
+]
+"""
+
+GOOD_JSON_DPORT = """
+[
+{
+    "ipv4-filter": {
+      "entry": [
+        {
+          "action": {
+            "accept": {}
+          },
+          "match": {
+            "protocol": 6,
+            "destination-port": { "value": 53 }
+          },
+          "sequence-id": 5
+        }
+      ]
+    }
+}
+]
+"""
+
+GOOD_JSON_MULTI_PROTO_DPORT = """
+[
+{
+    "ipv4-filter": {
+      "entry": [
+        {
+          "action": {
+            "accept": {}
+          },
+          "match": {
+            "protocol": 17,
+            "source-port": { "range": { "start": 1024, "end": 65535 } },
+            "destination-port": { "range": { "start": 1024, "end": 65535 } }
+          },
+          "sequence-id": 5
+        },
+        {
+          "action": {
+            "accept": {}
+          },
+          "match": {
+            "protocol": 6,
+            "source-port": { "range": { "start": 1024, "end": 65535 } },
+            "destination-port": { "range": { "start": 1024, "end": 65535 } }
+          },
+          "sequence-id": 10
+        }
+      ]
+    }
+}
+]
+"""
+
+GOOD_JSON_EVERYTHING = """
+[
+{
+    "ipv4-filter": {
+      "entry": [
+        {
+          "action": {
+            "accept": {}
+          },
+          "match": {
+            "protocol": 17,
+            "destination-ip": { "prefix": "10.2.3.4/32" },
+            "destination-port": { "value": 53 },
+            "source-ip": { "prefix": "10.2.3.4/32" }
+          },
+          "sequence-id": 5
+        },
+        {
+          "action": {
+            "accept": {}
+          },
+          "match": {
+            "protocol": 6,
+            "destination-ip": { "prefix": "10.2.3.4/32" },
+            "destination-port": { "value": 53 },
+            "source-ip": { "prefix": "10.2.3.4/32" }
+          },
+          "sequence-id": 10
+        }
+      ]
+    }
+}
+]
+"""
+GOOD_HEADER_INET6 = """
+header {
+  comment:: "The general policy comment."
+  target:: nokiasrl good-name-v6 inet6
+}
+"""
+
+# Print a info message when a term is set to expire in that many weeks.
+# This is normally passed from command line.
+EXP_INFO = 2
+
+TEST_IPS = [nacaddr.IP('10.2.3.4/32'), nacaddr.IP('2001:4860:8000::5/128')]
+
+
+_TERM_SOURCE_TAGS_LIMIT = 30
+_TERM_TARGET_TAGS_LIMIT = 70
+_TERM_PORTS_LIMIT = 256
+
+
+class NokiaSRLTest(absltest.TestCase):
+    def setUp(self):
+        super().setUp()
+        self.naming = mock.create_autospec(naming.Naming)
+
+    def _StripAclHeaders(self, acl):
+        return '\n'.join(
+            [line for line in str(acl).split('\n') if not line.lstrip().startswith('#')]
+        )
+
+    @capture.stdout
+    def testSaddr(self):
+        self.naming.GetNetAddr.return_value = TEST_IPS
+
+        acl = nokiasrl.NokiaSRLinux(
+            policy.ParsePolicy(GOOD_HEADER + GOOD_SADDR, self.naming), EXP_INFO
+        )
+        expected = json.loads(GOOD_JSON_SADDR)
+        self.assertEqual(expected, json.loads(str(acl)))
```

Comment:
The output is already checked with @capture.stdout so this assertion isn't strictly necessary imo.

Reply:
It's superfluous, I copied it from the OpenConfig tests. It's in all the methods - also in other parts.

e.g. https://github.com/aerleon/aerleon/blob/main/tests/regression/cisconx/cisconx_test.py#L239
is also checked by stdout capture.

Happy to remove them, but it may be better to do that for all tests in a separate commit/branch